examples/sfu-videochat/index.html

```diff
@@ -4,7 +4,7 @@
   <title>SkyWay - Video chat example</title>
   <link rel="stylesheet" href="style.css">
   <script type="text/javascript" src="//ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
-  <script type="text/javascript" src="/dist/skyway.js"></script>
+  <script type="text/javascript" src="//cdn.webrtc.ecl.ntt.com/skyway-latest.js"></script>
   <script type="text/javascript" src="/examples/key.js"></script>
```

Comment:
while we're fixing this, could you please make this a relative path so it doesn't matter what the root directory of the server is? 

i.e.
`../key.js`

Reply:
ok!